frontend/packages/topology/integration-tests/support/pages/topology/topology-page.ts

```diff
@@ -268,11 +268,10 @@ export const topologyPage = {
       .find(topologyPO.graph.groupLabel)
       .contains(serviceName);
   },
-  getKnativeRevision: (serviceName: string) => {
+  getKnativeRevision: (revisionName: string) => {
     return cy
-      .get('[data-type="knative-service"]')
-      .find('[data-type="knative-revision"] g.odc-workload-node')
-      .contains(serviceName);
+      .get('[data-type="knative-revision"]')
+      .find(`[data-test-id="${revisionName}"] g.odc-workload-node`);
```

Comment:
e2e is failing here looks like

Reply:
Thanks @invincibleJai, I can't get the local tests to run so I've been struggling.

Reply:
Thanks Jeff, waiting for it to get in 👍